src/utils.ts

```diff
@@ -34,3 +34,11 @@ export async function getSwaggerJson(options: IOptions): Promise<string> {
 export function sortBy<T>(fn: (value: T) => string): (a: T, b: T) => number {
     return (a: T, b: T) => (fn(a) || '').localeCompare(fn(b) || '');
 }
+
+export function copyFile(src: string, destination: string): void {
```

Comment:
we can use promise api directly (available sincle node 10)

import { promises: fs } from 'fs';

fs.copyFile('from', 'to');